uPortal-webapp/src/main/resources/properties/portal.properties

```diff
@@ -545,6 +545,11 @@ org.apereo.portal.portlet.container.services.PortletCookieServiceImpl.portalCook
 ##
 org.apereo.portal.rest.search.PortletsSearchStrategy.displayScore:true
 
+##
+## Flag to enable or disable the display of the favorite flag in the results
+##
+org.apereo.portal.rest.search.PortletsSearchStrategy.favoriteFlagScore:true
```

Comment:
Hmm, I set this to true but should it instead be 'false' by default?

Reply:
Since this is a new property I would be okay with it defaulting to `true`